roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -10,6 +10,30 @@
     - ansible_distribution == 'CentOS'
     - ansible_distribution_major_version == '7'
 
+- name: 'Install centos-release-ansible29 on CentOS8'
+  yum:
+    name: centos-release-ansible-29
+    state: latest
+    update_cache: true
+  tags:
+    - packages
+  when:
+    - ansible_distribution == 'CentOS'
+    - ansible_distribution_major_version == '8'
```

Comment:
I think a variable here to let users decide if they want the Ansible repository enabled would be good. As there may be users of this collection that do not need this repository present.

Reply:
Right! I'm exclusively working with Katello in my setups, forgot about that ;-)